mysql-test/suite/innodb_fts/t/phrase.test

```diff
@@ -0,0 +1,39 @@
+-- source include/have_innodb.inc
+
```

Comment:
We got [this ./mtr --ps-protocol --embedded failure](https://buildbot.mariadb.org/#/builders/536/builds/11170/steps/7/logs/stdio) that starts like this:
```diff
@@ -42,8 +42,8 @@
 Sort_merge_passes	0
 Sort_priority_queue_sorts	0
 Sort_range	0
-Sort_rows	2
-Sort_scan	1
+Sort_rows	4
+Sort_scan	2
 No sorting for this query even if MATCH is part of an expression
 FLUSH STATUS;
 SELECT title, MATCH(title, text) AGAINST ('database') AS score
```
I can reproduce this with `./mtr --ps-protocol --embedded` as well as `./mtr --ps-protocol` but not with `./mtr --embedded`. Maybe we just need `disable_ps_protocol` or `disable_ps2_protocol` somewhere in this test.

Reply:
Adding either `disable_ps2_protocol` or `disable_ps_protocol` would not fix this.

Reply:
I worked around this by adding the following:
```
if ($PS_PROTOCOL) { skip Counter differences for --ps-protocol; }
```